jina-embeddings/jina-embeddings-v2-base-en/model/model.py

```diff
@@ -0,0 +1,16 @@
+from transformers import AutoModel
+
+class Model:
+    def __init__(self, **kwargs):
+        self._model = None
+
+    def load(self):
+        self._model = AutoModel.from_pretrained(
```

Comment:
bonus, you can use `hf_cache` in the config with these settings so the cold start is fire out of the box.

Reply:
Added!